package.json

```diff
@@ -75,13 +76,15 @@
     "@bbc/psammead-media-player": "^2.7.11",
     "@bbc/psammead-most-read": "^4.1.4",
     "@bbc/psammead-navigation": "^6.0.11",
+    "@bbc/psammead-oembed": "0.1.0-alpha.1",
```

Comment:
@simonsinclair Do we still need the psammead oembed component? I know it's not being used anywhere at the moment.

Reply:
It's not needed. 🙂  
I discussed removing it with @jamesdonoh earlier in the year, after we went down another route with Social Embeds. We decided it wouldn't hurt to leave it, but I wouldn't be against it being removed – especially as it never came out of alpha and wouldn't be hard to build again.

Reply:
Cool, I'll remove it from here for now, you can remove the component itself later if you need to :)